.buildkite/scripts/serverless/create_deploy_tag/create_deploy_tag.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+DEPLOY_TAG="deploy@$(date +%s%3N)"
```

Comment:
Previously, the created tags were accurate to the closest second
```suggestion
DEPLOY_TAG="deploy@$(date +%s)"
```